File: Lista 9/L9_Q6.c
#include <stdio.h>

int main(void){
    int n_valor = 0, i, valor, r = 0;

    printf("Informe a quantidade de valores: ");
    scanf("%d", &n_valor);

    for(i = 1; i<= n_valor; i++)
    {
        printf("informe o %d valor: ", i);
        scanf(" %d", &valor);

        r = r + valor;
    }
    r = r / n_valor;

    printf("a media artimetica dos valores eh: %d", r);
}

File: Período_1/Lista 9/Python/L9_Q6.py
qntValores = int(input("Informe a quantidade de valores: "))
soma = 0
for i in range(qntValores):
    valor = float(input(f"Informe o {i + 1}º valor: "))

    soma += valor

media = soma / qntValores

print(f"A media aritmética dos valores lidos é {media}")